packages/wonder-blocks-dropdown/components/multi-select-menu.test.js

```diff
@@ -72,15 +80,20 @@ describe("MultiSelectMenu", () => {
     });
 
     it("selects items as expected", () => {
-        expect(menu.state("open")).toEqual(true);
+        menu.setState({open: true});
+        const noop = jest.fn();
+        const nativeEvent = {
+            nativeEvent: {stopImmediatePropagation: noop},
+        };
+
         expect(menu.prop("selectedValues")).toEqual(["2"]);
 
         // Grab the second item in the list
         const item = menu.find(SelectItem).at(0);
         expect(item.text()).toEqual("item 1");
         // Click the item 2, deselecting it
         item.simulate("mousedown");
-        item.simulate("mouseup");
+        item.simulate("mouseup", nativeEvent);
```

Comment:
Interesting.  I'm surprised `simulate` doesn't does this automatically.  Can you open an issue with `enzyme` and/or `react-dom/test-utils`, maybe they'll add stubs for it eventually?